.github/workflows/tsan.yaml

```
@@ -1,35 +1,39 @@
 name: tsan
-on: 
+on:
   pull_request:
   schedule:
     - cron: '7 0 * * *'
 
 jobs:
   tsan:
     name: tsan
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-jammy-latest
     steps:
       - name: deps
-        uses: ros-tooling/setup-ros@v0.2
+        uses: ros-tooling/setup-ros@v0.3
```

A new version has been released related to some problems in RMF https://github.com/ros-tooling/setup-ros/pull/507#pullrequestreview-1099653924
```suggestion
        uses: ros-tooling/setup-ros@v0.4
```